doc/benchmark.dtd

```diff
@@ -15,6 +15,7 @@
 
 <!ELEMENT option (#PCDATA)>
 <!ELEMENT propertyfile (#PCDATA)>
+<!ELEMENT multiproperty (#PCDATA)>
```

Comment:
Currently only value `multiproperty` is used for obtaining several statuses. By default attribute `kind` is not set.